js/generate.js

```diff
@@ -214,13 +221,19 @@ function fillTables() {
 			}
 			fillCell(el8, data.tests);
 			el8.classList.add("tests");
-                        if (counterReq == counterRes) {
+		        });
+                    };
+                }
+                var p = fetch("data/" + spec + ".json").then(
```

Comment:
> What does the text say?

Sorry, not sure what you are are asking here. 

Before proceeding, we should really wait for @dontcallmedom to comment on the use of `fetch()`. If code is to be compatible with today's browsers (or if this code is tested with PhantomJS), then using `fetch()` might be a show-stopper.